### DISCUSSION TITLE
x-on:click fires twice in cloned template

### DISCUSSION THREAD

**Discussion by ghost** [alpinejs/alpine#2255] [7. Bugs (archived)] [answered]
I don't know if this is related to [#348](https://github.com/alpinejs/alpine/issues/348).

here's a pen https://codepen.io/aeonelo/pen/ZEJBKQJ

**Comment by danddanddand** [comment, marked as answer]
It wouldn't be related to that ticket because that was V2.

I'm not at computer so can't play with it but you can try adding x-ignore to template and then use initTree after you clone it to initialize it. 

My other thought is just to use x-if because it basically does exactly what you're doing with a lot less code.

**Reply by ghost** [reply]
Yo! `x-ignore` did the trick. Thanks!
